Runtime/Behaviours/CinemachineStoryboard.cs

```diff
@@ -251,7 +251,7 @@ void PlaceImage(CanvasInfo ci, float alpha)
                 ci.mViewport.localRotation = Quaternion.identity;
                 ci.mViewport.localScale = Vector3.one;
                 ci.mViewport.ForceUpdateRectTransforms();
-                ci.mViewport.sizeDelta = new Vector2(screen.width - Mathf.Abs(wipeAmount), screen.height);
+                ci.mViewport.sizeDelta = new Vector2(screen.width + 1 - Mathf.Abs(wipeAmount), screen.height + 1);
```

Comment:
Can you prove to me that precisely-sized images won't get stretched?  i.e. we can do pixel-perfect storyboards